src/remote.go

```diff
@@ -66,7 +66,8 @@ const (
 var (
 	ErrPathNotExists                  = errors.New("remote path doesn't exist")
 	ErrNetLookup                      = errors.New("net lookup failed")
-	ErrClashesDetected                = fmt.Errorf("clashes detected. use `%s` to override this behavior", CLIOptionIgnoreNameClashes)
+	ErrClashesDetected                = fmt.Errorf("Clashes detected. Use `%s` to override this behavior or `%s` to try fixing this",
```

Comment:
When you compose multiple errors on the same line, e.g. "pushErr: %v indexErr: %v" then that's when capitalizing the first letter becomes awkward.
Please revert the first letter to what it was "c" vs "C"